invenio_rdm_records/resources/resources.py

```diff
@@ -66,6 +68,16 @@ def p(route):
             route("PUT", p(routes["item-review"]), self.review_update),
             route("DELETE", p(routes["item-review"]), self.review_delete),
             route("POST", p(routes["item-actions-review"]), self.review_submit),
+            route(
+                "POST",
+                p(routes["user-access-request"]),
+                self.create_user_access_request,
+            ),
+            route(
+                "POST",
+                p(routes["guest-access-request"]),
+                self.create_guest_access_request_token,
+            ),
```

Comment:
Minor: IMHO I would move these 2 routes to the `RDMRecordRequestsResource`?

Reply:
The `RDMRecordRequestsResource` uses the `RecordRequestsService`, which doesn't have the necessary methods (`create_user_access_request`, `create_guest_access_request_token`) available.
These are part of the `RecordAccessService`, which is available as sub-service `current_rdm_records_service.access`.